### ISSUE TITLE
Editors cannot edit response letter to reviewers

### ISSUE THREAD

**Issue by cpence** [openlibhums/janeway#4290]
**Describe the bug**
As far as I can see, there is no way for editors to edit the text that an author submits with their revision in the "Response Letter to Reviewers" box. This could be important if, for instance, an author accidentally reveals their identity in this text box, which will be shared with future rounds of peer reviewers.

**Janeway version**
Unknown – Michigan Press Hosted

**To Reproduce**
1. Visit a paper in Review phase with a completed revision request.
2. Under "Revision Requests," click "View."
3. Nowhere on this page allows for an editor to change the text in the right-hand column, "Response Letter to Reviewers."

**Expected behavior**
Editors and other administrators should have access to edit this text, as they do with almost all other such texts in the review process.

**Comment by cpence** [comment]
I should also note that, as far as we can tell, there is no way for authors to edit their response-to-reviewer letter either, once it has been uploaded. (We tried most all combinations of open/closed rounds of review, etc.) This also seems to be something that the author should be allowed to change, in case of a mistake?
